orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
yea, I'm totally fine with that - but there's no reason for it to have a type other than `string` in that case.

I was just trying to make the point that defining the `CacheType` type wasn't adding anything useful.